src/Admin/AbstractAdmin.php

```diff
@@ -352,7 +352,7 @@ abstract class AbstractAdmin extends AbstractTaggedAdmin implements AdminInterfa
     public function getExportFormats(): array
     {
         return [
-            'json', 'xml', 'csv', 'xls',
+            'json', 'xml', 'csv', 'xls', 'xlsx',
```

Comment:
If you do this, `xlsx` will always be added, even if the spreadsheet library is not installed.

Looking at https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Bridge/Exporter/AdminExporter.php#L45-L53, if we return `[]`, the available formats default to the one provided by the exporter. So I wonder if changing this array to `[]` wouldn't be the best way to do what we want.

Could you try on your project ?

Reply:
I agree. I tested it and if I return [] there, it will use the default_writers of the exporter, unless you override this method. I will change it.